requirements.txt

```diff
@@ -1,7 +1,6 @@
 asyncpg
-backoff
```

Comment:
not bothering to maintain

Reply:
in a follow-up pr i think i will try/catch the whole process & return an err msg to the discord if anything goes wrong